src/dispatcher/src/main/java/org/acme/ControllerResource.java

```diff
@@ -0,0 +1,51 @@
+package org.acme;
```

Comment:
Could you rename packages following the same standard used in the loader app, e.g. "com.redhat.knative.demo.loader"?